.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Set up node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --immutable
```

Comment:
There is no need to `yarn install` since there are no deps and there is no `yarn lint`, you can remove this file at all